app/src/main/res/values/strings.xml

```diff
@@ -233,43 +233,43 @@
     <string name="skin_fruity">Fruity</string>
     <string name="skin_transparent">Transparent</string>
 
-    <string name="block_popup">Block Popup</string>
-    <string name="open_window_hint">Website try to open a new window</string>
+    <string name="block_popup">Block Pop-up</string>
+    <string name="open_window_hint">Website is trying to open a new window</string>
 
     <string name="action_save_web_page">Save Web Page</string>
     <string name="action_saved_pages">Saved Pages</string>
-    <string name="saved_page_successfully">Saved Page Successfully, It is in the menu of homepage: Saved Pages</string>
-    <string name="deleted_failed">"Deleted Failed. The file located in download path: "</string>
+    <string name="saved_page_successfully">Page Saved Successfully. It is in the menu of homepage: Saved Pages</string>
+    <string name="deleted_failed">"Failed to Delete. The file located in download path: "</string>
 
     <string name="empty_hint">Nothing to show&#8230;</string>
 
     <string name="network_log">Network Log</string>
     <string name="catalog">Catalog</string>
 
-    <string name="screenshot_hint">There may be compatibility issues in android 5.0+ to take a full screenshot of website. We are still looking for solutions.</string>
+    <string name="screenshot_hint">There may be compatibility issues in Android 5.0+ to take a full screenshot of website. We are still looking for solutions.</string>
 
     <string name="conn_timeout">Connection timed out</string>
 
-    <string name="already_default">It already be set to default</string>
+    <string name="already_default">It has already been set to default</string>
 
-    <string name="set_to_default">It will be set to default after using it open link(%s), please choose via to open the link.</string>
+    <string name="set_to_default">It will be set to default after opening link(%s), please choose via to open the link.</string>
 
-    <string name="quick_back_desc">There may be compatibility issues when you open this switch. You should close it when you found going back work not well.</string>
+    <string name="quick_back_desc">There may be compatibility issues when you enable this option. You should disable it when you find going back not work well.</string>
```

Comment:
Could you change this to (Or at least more like it) There may be compatibility issues when you enable this option. You should disable it if you find going back working poorly.